voxblox_ros/src/esdf_server.cc

```diff
@@ -112,12 +114,10 @@ void EsdfServer::updateMesh() {
     publishTraversable();
   }
 
-  if (publish_esdf_map_ && esdf_map_pub_.getNumSubscribers() > 0) {
-    const bool only_updated = false;
-    voxblox_msgs::Layer layer_msg;
-    serializeLayerAsMsg<EsdfVoxel>(esdf_map_->getEsdfLayer(), only_updated,
-                                   &layer_msg);
-    esdf_map_pub_.publish(layer_msg);
+  // VERY important to call this before calling the mesh update, as this clears
```

Comment:
Should these be seperate flags then?

Reply:
Aah this is a left-over comment... I abandoned the incremental esdf update stuff for now (was getting to be too big of a mess, might require a bigger refactor). It actually *is* different flags, since esdf blocks have their own "updated" field. It's already a bit of a mess with the TSDF block updated field, since both the esdf update and the mesh update share the one flag. It might make sense to do a bitmask or something to be honest. But yeah, I'll remove this comment and we should discuss how to best handle this.